src/app/components/MediaLoader/configs/tvMedia.ts

```diff
@@ -0,0 +1,49 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: tvMediaBlock } = filterForBlockType(blocks, 'tvMedia');
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: tvMediaBlock.episodeTitle,
+    duration: tvMediaBlock.versions[0].duration,
+    durationISO8601: tvMediaBlock.versions[0].durationISO8601,
+    type: 'video',
+    holdingImageURL: `https://${tvMediaBlock.imageUrl}`,
```

Comment:
I had a look at the other clipMedia / aresMedia files, and they seem to use the `buildIChefUrl` function to create the holding image. I think we need to do the same here, so that the holding image is correct. See https://github.com/bbc/simorgh/blob/227b97d6bbb133584e84dc75b7e7987089989438/src/app/components/MediaLoader/configs/clipMedia.ts#L60-L65

Reply:
Also, let's extract `holdingImageURL` into a separate variable, and use it in multiple places below.